python_modules/libraries/dagster-dg/dagster_dg/cli/docs.py

```diff
@@ -101,3 +105,35 @@ def serve_docs_command(
 
         finally:
             yarn_dev.terminate()  # pyright: ignore[reportPossiblyUnboundVariable]
+
+
+@docs_group.command(name="build", cls=DgClickCommand)
+@click.argument("output_dir", type=click.Path(exists=False))
+@dg_global_options
+@cli_telemetry_wrapper
+def build_docs_command(
```

Comment:
is this something an end user would use? if not should it be hidden?

Reply:
it is, they can use it to build a static webapp (to host themselves)